designer/server/plugins/routes/newConfig.ts

```diff
@@ -40,7 +41,16 @@ export const registerNewFormWithRunner: ServerRoute = {
           await publish(newName, copied);
         }
       } catch (e) {
+        consoleMessages.publishError();
         request.logger.error(e);
+        return h
+          .response(
+            `Designer could not connect to runner instance. 
+Please confirm it is up and running, 
+and in development mode. See docs for more details.`
+          )
+          .type("application/json")
```

Comment:
should the response be an object/json if the type is application/json? although like I mention above -- this shouldn't be exposed to a form designer
```
{ message: "..."} 
```

Reply:
Done